src/assembler/encoder.ts

```diff
@@ -237,19 +237,53 @@ function writeDataInstruction(
   }
   let bytes: Byte[] = []
   let alignment: number = 0
-  const values = instruction.options.map((x) => +x)
+  let optionsContainAString: boolean = false
+  let values: number[] = []
+  instruction.options.forEach((x) => {
+    if (x.startsWith('"')) {
+      optionsContainAString = true
+      x = x.substring(0, x.length - 1).substring(1)
+      const charArray: string[] = x.split('')
```

Comment:
instead of making char array. use charCodeAt with i as param and i+1 for escapted "